### PR TITLE
Add rule to detect presence of secp256k1 private keys

### PR DESCRIPTION
**Pull request by mtsalenc**
Black hat hackers have bots continuously monitoring github for accidental inclusion of private keys in commits and immediatly attempt to steal funds secured by it on blockchains.

This PR adds a rule to check against the inclusion of secp256k1 private keys, used by the Bitcoin and Ethereum blockchains.

### CODE REVIEW

File: packages/@secretlint/secretlint-rule-secp256k1-priv/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
+import { SecretLintRuleCreator, SecretLintSourceCode } from "@secretlint/types";
+
+import secp256k1 from 'secp256k1'
+import BN from 'bn.js'
+
+export const messages = {
+    secp256k1Priv: {
+        en: "found secp256k1 private key: {{KEY}}",
+        ja: "secp256k1 秘密鍵 {{KEY}} が見つかりました" // TODO: Review needed.
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-secp256k1-priv/src/index.ts]:
```suggestion
        ja: "秘密鍵(secp256k1) {{KEY}} が見つかりました"
```

File: packages/@secretlint/secretlint-rule-secp256k1-priv/package.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    "homepage": "https://github.com/secretlint/secretlint/tree/master/packages/@secretlint/secretlint-rule-secp256k1-priv/",
+    "bugs": {
+        "url": "https://github.com/secretlint/secretlint/issues"
+    },
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/secretlint/secretlint.git"
+    },
+    "license": "MIT",
+    "author": "azu",
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-secp256k1-priv/package.json]:
```suggestion
    "author": "mtsalenc",
```

File: packages/@secretlint/secretlint-rule-secp256k1-priv/README.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+Install with [npm](https://www.npmjs.com/):
+
+    npm install @secretlint/secretlint-rule-secp256k1-priv
+
+
+## MessageIDs
+
+### secp256k1Priv
+> found secp256k1 private key: {{KEY}}
+
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-secp256k1-priv/README.md]: Can you add short description for this message key.

For example,

> Secp256k1 curve private keys such as used by Bitcoin an Ethereum.
> It should be treated as credentials.

File: packages/@secretlint/secretlint-rule-secp256k1-priv/CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Changelog
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-secp256k1-priv/CHANGELOG.md]: We will create CHANGELOG file when publish this package automatically by CI.
I don't need you to create the CHANGELOG file.

https://github.com/secretlint/secretlint/blob/a5f56d1df9c9d88b7a8ff82867ab7e51aab06a20/.github/workflows/publish.yml#L53

File: packages/@secretlint/secretlint-rule-secp256k1-priv/README.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+# @secretlint/secretlint-rule-secp256k1-priv
```
Comment (azu) [inline packages/@secretlint/secretlint-rule-secp256k1-priv/README.md]: I prefer to use `privatekey` than `priv`.

```suggestion
# @secretlint/secretlint-rule-secp256k1-privatekey
```

**Comment by mtsalenc** [comment]
I've implemented the suggested changes. Please, check again.

**Pull request merged**